bin/seeds.js

```diff
@@ -0,0 +1,107 @@
+// To insert in "bin/seeds.js"
+const mongoose = require("mongoose");
+const DB_NAME = "movie-project";
```

Comment:
this variable was here to be added on the mongoose connection url in line 5, but if the name of the database is there as a string, then this variable is not needed